python/pfs/drp/stella/buildFiberProfiles.py

```diff
@@ -268,9 +268,9 @@ def associatePeaks(self, peaks, width):
                 del indices[-1]
                 unassociated[pp.peak] = pp
                 for index in indices:
-                    # Score is distance to most recent trace peak + negative number of past associations
+                    # Score is negative number of past associations and distance to most recent trace peak
```

Comment:
This comment is a little confusing as the ``score`` variable itself is till the distance to the most recent trace peak. Suggest renaming the variable.